deployments/mainnet/weth/migrations/1675148548_set_up_ens_subdomain_and_text_record.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { proposal } from '../../../../src/deploy';
+
+interface Vars {}
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = ENSSubdomainLabel + '.' + ENSName;
+// XXX can fetch this programmatically from the ENS registry
```

Comment:
The subdomain?

Reply:
Ah. Is the XXX bc you are wondering if we should do that?

Reply:
Yeah, I'll remove the XXX for now to avoid any confusion